dsp/modules/sentence_vectorizer.py

```diff
@@ -24,28 +25,29 @@ def __call__(self, inp_examples: List["Example"]) -> np.ndarray:
 
     def _extract_text_from_examples(self, inp_examples: List) -> List[str]:
         if isinstance(inp_examples[0], str):
-            return inp_examples 
+            return inp_examples
         return [" ".join([example[key] for key in example._input_keys]) for example in inp_examples]
 
 
 class SentenceTransformersVectorizer(BaseSentenceVectorizer):
-    '''
+    """
```

Comment:
can I ask for reverting these changes in the other classes?

we have a plan to fix them all in one PR to keep the git blame cleaner.